test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -152,6 +152,18 @@ def test_non_integer_timestamp
     end
   end
 
+  # The conversions from user input to output.
+  def latency_conversion
+    # TODO(lingshi): Use http://ruby-doc.org/core-2.4.0/Hash.html#method-i-transform_values
+    # instead when we upgrade to ruby 2.4.0.
+    # super.transform_values do |expected|
+    #   expected.reject { |_, value| value == 0 }
+    # end
+    super.map do |input, expected|
+      [input, expected.reject { |_, value| value == 0 }]
```

Comment:
Does the REST path require us to send 'nanos' => 0? If not, can we just get rid of those throughout?